### PR TITLE
Added testing to list of change types

### PR DESCRIPTION
**Pull request by willemwe0**
## Proposed changes

_Description of the changes implemented by this PR, feel free to be as verbose as needed_

...

## Types of changes

What type of changes does your code introduce?
_Put an `x` in the boxes that apply, leave blank if none_

- [ ] Frontend
- [ ] Backend
- [ ] Bugfix
- [ ] New feature
- [ ] Component of Major Release
- [ ] Project Management
- [ ] Quality of Life

## Checklist

_Put an `x` in the boxes that apply._

- [ ] Local Cypress Tests Pass with my changes (`yarn cypress:run`)
- [ ] Lint and unit tests pass locally with my changes (`yarn lint && ./vendor/bin/phpunit`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

## Dependant PRs

_List Pull Requests below in format `#<ISSUE-NUMBER>`_
- ...

## Relevant Issues

_List Issues below in format `#<ISSUE-NUMBER> [Full|Partial]`_
- ...

## Description

_If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc._

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ _Put an `x` in the boxes that apply, leave blank if none_
 - [ ] Component of Major Release
 - [ ] Project Management
 - [ ] Quality of Life
+- [ ] Improving/Fixing/Adding Tests
```
Comment (robguy21) [inline PULL_REQUEST_TEMPLATE.md]: What if we called this- "Test Improvements"

Comment (willemwe0) [reply PULL_REQUEST_TEMPLATE.md]: @robguy21 done

**Review by robguy21 — APPROVED** [review]

**Review by countnoobula — APPROVED** [review]

**Review by pruhesi — APPROVED** [review]

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -16,6 +16,7 @@ _Put an `x` in the boxes that apply, leave blank if none_
 - [ ] Component of Major Release
 - [ ] Project Management
 - [ ] Quality of Life
+- [ ] Test Improvements
```
Comment (sscholle) [inline PULL_REQUEST_TEMPLATE.md:19]: does this not fall under `code quality` or `quality of life`?

**Review by robguy21 — APPROVED** [review]
community approved!

**Review by sscholle — APPROVED** [review]

**Review by musangowope — APPROVED** [review]

**Review by adameier — APPROVED** [review]

**Pull request merged**